DataConnectors/GCP/ArmTemplate/GCPOnboarding.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "DataCollectionEndpoint_name": {
+      "defaultValue": "DataCollectionEndpoint",
+      "type": "String"
+    },
+    "Location": {
+      "defaultValue": "[resourceGroup().location]",
+      "type": "String"
+    },
+    "DataCollectionRule_name": {
+      "defaultValue": "GCPAuditLogsDataCollectionRule",
+      "type": "String"
+    },
+    "workspace_name": {
+      "defaultValue": "",
+      "type": "String"
+    }
+  },
+  "variables": {},
+  "resources": [
+    {
+      "type": "Microsoft.Insights/dataCollectionEndpoints",
+      "apiVersion": "2021-09-01-preview",
+      "name": "[parameters('DataCollectionEndpoint_name')]",
+      "location": "[parameters('Location')]",
+      "properties": {
+        "networkAcls": {
+          "publicNetworkAccess": "Enabled"
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Insights/dataCollectionRules",
+      "apiVersion": "2021-09-01-preview",
+      "name": "[parameters('DataCollectionRule_name')]",
+      "location": "[parameters('Location')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Insights/dataCollectionEndpoints',parameters('DataCollectionEndpoint_name'))]"
+      ],
+      "properties": {
+        "dataCollectionEndpointId": "[resourceId('Microsoft.Insights/dataCollectionEndpoints',parameters('DataCollectionEndpoint_name'))]",
+        "destinations": {
+          "logAnalytics": [
+            {
+              "workspaceResourceId": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.OperationalInsights/workspaces/', parameters('workspace_name'))]",
```

Comment:
i prefer to create the workspace resource id as an arm variable and use it here like they say here: https://stackoverflow.com/questions/64803341/concat-variable-with-string-in-azure-arm-template